### DISCUSSION TITLE
Enquiry about Facial Format Identifier

### DISCUSSION THREAD

**Discussion by yeelongchong** [GitbookIO/community#239] [0. General bugs and questions] [answered]
### Ask a question, start a conversation, or make an annoucement

In https://docs.mosip.io/1.1.5/biometrics/biometric-specification, it is using ISO/IEC 19794-5:2011 for face, format identifier is FAC.
In https://docs.mosip.io/1.2.0/biometrics/abis-api, it mentioned FID instead of FAC for facial format identifier.

Can anyone help to clarify above?

Thanks.

**Comment by aikoii** [comment, marked as answer]
Heya! Looks like you can reach out to MOSIP's community on their [Gitter Channel](https://gitter.im/mosip-community/community). They should be able to help you out 🙂 🤞
